### PR TITLE
hide_secrets: fieldname should not be multline or a series of words

### PR DESCRIPTION
**Pull request by goneri**
Ensure we don't match a multiline block of text as a field name just because it
happens to contain `host` and `key`.

### CODE REVIEW

File: ansible_anonymizer/anonymizer.py [hunk 1]
```diff
@@ -187,7 +199,7 @@ def anonymize(o: Any, key_name: str = "") -> Any:
 
 
 def hide_secrets(block: str) -> str:
-    flags = re.MULTILINE | re.DOTALL | re.IGNORECASE
```
Comment (TamiTakamiya) [inline ansible_anonymizer/anonymizer.py:190]: 👍

File: ansible_anonymizer/anonymizer.py [hunk 2]
```diff
@@ -85,6 +85,18 @@ def is_password_field_name(name: str) -> bool:
     return re.search(DENYLIST_REGEX_WITH_PREFIX, name) is not None
 
 
+def is_jinja2_expression(value: str, quoted: bool = True) -> bool:
+    if quoted:
+        if re.match(r"^\s*\"\s*{{\s*.*?\s*}}\s*\"\s*$", value):
```
Comment (TamiTakamiya) [inline ansible_anonymizer/anonymizer.py:90]: Though I am not an expert of regex and it won't make any differences, isn't `{{\s*.*?\s*}}` equivalent to `{{.*?}}`?

**Review by TamiTakamiya — APPROVED** [review]
LGTM. Just left one minor comment. Thanks!

File: ansible_anonymizer/anonymizer.py (hunk 2 above)
Comment (goneri) [reply ansible_anonymizer/anonymizer.py:90]: Indeed, the `\s` are superfluous. I will right this differently.

**Pull request merged**